tcg-useffect/src/App.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect,useState } from 'react';
+import './App.css';
+
+function App() {
+  const [count, setCount] = useState(0);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setCount((count) => count + 1);
```

Comment:
Okay, great. The idea was to have this as the only change, for example:

https://github.com/kylemeenehan/tcg-useeffect/pull/1

But you can go ahead and merge this, and I'll send further exercises in Slack